meeting_2/what_type_is_this.py

```diff
@@ -3,8 +3,9 @@ def what_type_is_this(variable):
     what_type_is_this.line += 1
     print(f'line {what_type_is_this.line}: {variable} is ', end='')
```

Comment:
### Question

1. Why there is `f` in front of the string?
2. Why we need `end=''` as the second argument?

Reply:
Please put your answers here!

Reply:
1. f = format, equivalent to str.format()
2. end='' prevents a new line created after a print() statement

Reply:
@jjiajian 
Note: you could use ` to quote the code like thing in markdown
[Here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a markdown cheat sheet from Github you could check.